tests/test_controller.py

```diff
@@ -32,16 +32,13 @@ def test_registry(controller, decentralizedController):
     with raises(TransactionFailed): decentralizedController.assertOnlySpecifiedCaller(tester.a2, key2, sender = tester.k0)
     with raises(TransactionFailed): controller.assertOnlySpecifiedCaller(tester.a2, key2, sender = tester.k2)
 
-def test_suicideTokens(contractsFixture, controller, decentralizedController, controllerUser):
-    legacyRepContract = contractsFixture.contracts['LegacyRepContract']
-    legacyRepContract.faucet(0)
-    legacyRepContract.transfer(controllerUser.address, 40 * 10**18)
-    with raises(TransactionFailed): controller.suicideFunds(controllerUser.address, tester.a0, [legacyRepContract.address], sender = tester.k2)
+def test_suicide(contractsFixture, controller, decentralizedController, controllerUser):
```

Comment:
@joeykrug I now know why these tests were passing before and why it was correct.  These tests are meant to test `Controller`, not `Controlled`.  I have added some tests for testing `Controlled` and adjusted these tests to focus on just `Controller`.  The test solidity file intentionally implements `IControlled` rather than `Controlled` because it is meant to act as a mock for testing, not as a legitimate `Controlled`.  Meanwhile, over in `Controlled`, you'll notice that the tests don't depend on `Controller` and instead just `IController`.